files/fr/web/javascript/reference/operators/unsigned_right_shift_assignment/index.md

```diff
@@ -12,7 +12,7 @@ browser-compat: javascript.operators.unsigned_right_shift_assignment
 
 {{jsSidebar("Operators")}}
 
-L'opérateur de décalage à droite non signé et d'affectation (_`>>>=`_) décale la séquence de bits fournie par l'opérande gauche vers la droite, d'autant de bits qu'indiqués par l'opérande droit, puis affecte le résultat de l'opération à la variable indiquée par l'opérande gauche.
+L'**opérateur de décalage à droite non signé et d'affectation (`>>>=`)** décale la séquence de bits fournie par l'opérande gauche vers la droite, d'autant de bits qu'indiqués par l'opérande droit, puis affecte le résultat de l'opération à la variable indiquée par l'opérande gauche.
```

Comment:
The closing `**` should be after *affectation* and before the operator with code formatting (`>>>=`)

```suggestion
L'**opérateur de décalage à droite non signé et d'affectation** (`>>>=`) décale la séquence de bits fournie par l'opérande gauche vers la droite, d'autant de bits qu'indiqués par l'opérande droit, puis affecte le résultat de l'opération à la variable indiquée par l'opérande gauche.
```